templates/bet/utils/farcaster.ts

```
@@ -0,0 +1,35 @@
+        headers: {
+            accept: 'application/json',
+            api_key: process.env.NEYNAR_API_KEY!,
+            'content-type': 'application/json',
+        },
+    }
+
+    const r = (await fetch(`${NEYNAR_API_URL}/farcaster/user/search?q=${username}&limit=1`, options)
+        .then((response) => response.json())
+        .catch((err) => {
+            console.error('Failed to fetch user data by Farcaster username:', err)
+            throw new Error(`Invalid Farcaster username`)
```

Biome

[error] Do not use template literals if interpolation and special-character handling are not needed.

Unsafe fix: Replace with string literal


(lint/style/noUnusedTemplateLiteral)